src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -63,14 +64,24 @@ def load_arguments(self, _):
         c.argument('top', arg_type=top_arg_type)
         c.argument('all_', options_list=['--all'], action='store_true', help="List all items.")
         c.argument('fields', arg_type=fields_arg_type)
+        c.argument('sku', help='The sku of App Configuration', arg_type=get_enum_type(['free', 'standard']))
 
     with self.argument_context('appconfig create') as c:
         c.argument('location', options_list=['--location', '-l'], arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
-        c.ignore('sku')
+        c.argument('assign_identity', nargs='*', validator=validate_assigned_identity, arg_group='Managed Service Identity', is_preview=True, help="Accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity, or a resource id to refer user assigned identity. Check out help for more examples")
 
     with self.argument_context('appconfig update') as c:
         c.argument('tags', arg_type=tags_type)
 
+    with self.argument_context('appconfig identity') as c:
```

Comment:
Also add params and help for appconfig identity show

Reply:
all commands under "appconfig identity" command group will inherit self.argument_context('appconfig identity'), similarly it will also inherit  self.argument_context('appconfig'). Since identity show doesn't introduce arguments with different specification, we don't need to specify it here. 

---
In reply to: [364511385](https://github.com/Azure/azure-cli/pull/11795#discussion_r364511385) [](ancestors = 364511385)